### DISCUSSION TITLE
Configuring keymaps

### DISCUSSION THREAD

**Discussion by iamshm** [LazyVim/LazyVim#2320] [Q&A] [answered]
### Which key

There are few keymaps which I 'd like to change

like I want to move `<leader>c` for lsp options to `<leader>l`
but it has many child keymaps like  `<leader>ca` ,  `<leader>cA`  ,  `<leader>cF` ,  `<leader>cd` , `<leader>cr` 

And there are many such which key bindings, do we change them individually at each place or is there any better alternative 

### Comment

I want to change gcc to `<leader>/` but nowhere I am able to find this

### Neotreee

I want to open my files with `<C-v>` & `<C-x>` like telescope

I went ahead and search for this keymaps in the LazyVim git but i was unable to get my way around.

### Misc

1. What is the top thing that showing at what code level I am? How can I disable it?
<img width="800" alt="Screenshot 2024-01-06 at 9 18 46 PM" src="https://github.com/LazyVim/LazyVim/assets/43462941/e5340390-a19a-4505-9f9e-cfae55b6e192">

2. Also, making the colorscheme transparent leads to an error from noice saying no bg color found defaulting to #000000 how can I silence this

```
Highlight group 'NotifyBackground' has no background highlight
Please provide an RGB hex value or highlight group with a background value for 'background_colour' option.
This is the colour that will be used for 100% transparency.

require("notify").setup({
  background_colour = "#000000",
})

Defaulting to #000000
```

### P.S
Lastly, is there any discord or reddit, where we can have such discussions 
@folke thanks for one of the best nvim configurations

**Comment by dpetka2001** [comment, marked as answer]
### Which-key
The standard way right now is to change each and every keymap like described [here](https://www.lazyvim.org/plugins/lsp#%EF%B8%8F-customizing-lsp-keymaps). There's an effort from a user of this community to make a plugin that you can use to change the `<leader>` keys (so something from `<leader>c` to `<leader>l` like in your case, but only for `<leader>` keys). The plugin is [lazyvim-menu-addon](https://github.com/abeldekat/lazyvim-menu-addon). You might want to check it out and test it to see if it fits your needs.

### Comment
This keymap is being defined internally in [mini.comment](https://github.com/echasnovski/mini.comment), so you would have to change it in the plugin spec.

### Neo-tree
You would have to change the mappings in the plugin spec itself like in #558.

### Misc
The plugin that shows you the code level you're at is [nvim-treesitter-context](https://www.lazyvim.org/plugins/treesitter#nvim-treesitter-context). To disable it you just have to put in the plugin spec `enabled = false`.
No idea about the transparency, I don't use it.

**Comment by iamshm** [comment]
Hey thanks for such a quick response. I will go through those and ping you back

**Reply by dpetka2001** [reply]
I also answered you about `<Tab>/<S-Tab>` in one of the other threads. It should work.

**Comment by abeldekat** [comment]
@dpetka2001 
Thanks for suggesting the plugin!

I added a video demo to the [readme](https://github.com/abeldekat/lazyvim-menu-addon)

@iamshm,

In case you used the plugin, I would be very interested in your feedback.

**Reply by iamshm** [reply]
@abeldekat I will definitely give this a try around the weekend. Thanks for helping the commmunity. 🔥
